pkg/model/keycloak_deployment.go

```diff
@@ -256,6 +256,8 @@ func KeycloakDeployment(cr *v1alpha1.Keycloak, dbSecret *v1.Secret, dbSSLSecret
 						},
 					},
 					ServiceAccountName: getServiceAccountName(cr),
+					Tolerations:        cr.Spec.KeycloakDeploymentSpec.Tolerations,
+					NodeSelector:       cr.Spec.KeycloakDeploymentSpec.NodeSelector,
```

Comment:
Could you please add similar change to:
https://github.com/keycloak/keycloak-operator/blob/1b9a02eb2417e9124ebbf4e9f37ec415d25c6b34/pkg/model/keycloak_deployment.go#L281